llvm-external-projects/iree-dialects/python/IREEDialectsModule.cpp

```diff
@@ -63,6 +93,37 @@ PYBIND11_MODULE(_ireeDialects, m) {
   //===--------------------------------------------------------------------===//
   auto iree_pydm_m = m.def_submodule("iree_pydm");
 
+  py::class_<PyIREEPyDMSourceBundle>(
+      iree_pydm_m, "SourceBundle", py::module_local(),
```

Comment:
nit: especially with the namespace aliases I'd prefer if these were spelled out (`iree_pydm_module`) - seeing things like `pydm_d` and `iree_pydm_m` makes the code harder to skim without having the decoder ring in your head and look more like typos

Reply:
I'll take this as feedback too. This file is due for a split, at which point the need to parse such things. Really we shouldn't be defining multiple submodules in the same file, but it has made it easier to get started.